Scripts/BuildEngine.ps1

```diff
@@ -171,11 +169,7 @@ function Build([string]$configuration, [int]$VsVersion , [bool]$runBuild) {
             $cMakeCacheVariableOverride += ' -DCMAKE_CONFIGURATION_TYPES=Debug;Release '
         }
         "Linux" {
-            $cMakeGenerator = "-G `"Unix Makefiles`""
-
-            # Set Linux build compiler
-            $env:CC = '/usr/bin/gcc-11'
-            $env:CXX = '/usr/bin/g++-11'
+            $cMakeGenerator = "-G `"Ninja`""
```

Comment:
we want to update the README to provide installation steps for Ninja  on linux

Ideally we might want to automate that installation so devs wont need to worry about it.  we can do that in follow-up PR - please create an issue to track it